instrumentation/jboss-logmanager/jboss-logmanager-appender-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jbosslogmanager/appender/v1_1/LoggingEventMapper.java

```diff
@@ -81,13 +82,15 @@ public void capture(Logger logger, ExtLogRecord record) {
 
     Throwable throwable = record.getThrown();
     if (throwable != null) {
-      // TODO (trask) extract method for recording exception into
-      // io.opentelemetry:opentelemetry-api
-      attributes.put(ExceptionAttributes.EXCEPTION_TYPE, throwable.getClass().getName());
-      attributes.put(ExceptionAttributes.EXCEPTION_MESSAGE, throwable.getMessage());
-      StringWriter writer = new StringWriter();
-      throwable.printStackTrace(new PrintWriter(writer));
-      attributes.put(ExceptionAttributes.EXCEPTION_STACKTRACE, writer.toString());
+      if (builder instanceof ExtendedLogRecordBuilder) {
```

Comment:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/conventions/src/main/kotlin/io.opentelemetry.instrumentation.library-instrumentation.gradle.kts adds dependency to instrumentation-api-incubator that pulls in opentelemetry-api-incubator so `ExtendedLogRecordBuilder` is always present. Really this instrumentation doesn't use the instrumentation api.